lib/state/actions/home_page_actions.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:async';
+import 'package:counter_async_redux/api/handlers/pokemon_handler.dart';
+import 'package:counter_async_redux/state/actions/actions.dart';
+import 'package:counter_async_redux/state/app_state.dart';
+
+class GetPokemonList extends LoadingAction {
```

Comment:
can you follow this hierarchy for constructor and variables

`Class`
`     static variables`
`     constructor`
`     final variables`